lib/computer.ex

```diff
@@ -4,6 +4,65 @@ defmodule Computer do
   def move(board, space) do
     Board.make_mark(board, space, %Computer{}.mark)
   end
-  
-  
+
+  def get_best_move(game, depth \\ 0, best_score \\ []) do
+    cond do
+      Game.game_over?(game) -> 
+        score(game.board)
+      depth == length(Board.available_spaces(game.board)) ->
+        best_move(best_score)
+    end
+  end
+
+  def get_best_move(game, depth, best_score) do
+    Enum.map(Board.available_spaces(game.board), fn(space) ->
```

Comment:
I'd love a little insight into what's going on here if you have a minute :)

Reply:
+1